### DISCUSSION TITLE
Welcome to ha-fusion Discussions!

### DISCUSSION THREAD

**Discussion by matt8707** [matt8707/ha-fusion#115] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dos1973** [comment]
Hi, im very impressed and hyped from Fusion.

im a long Time IoBroker User and a pure Homeasistant Starter at all, 
the Tablet Dashboard Design was always i "dont like" to go with Home assistant. But Fusion changed my mind!

Thanks to all developers and contributers to build and make this possible!
- i will raise a lot of Wishes - :-)

**Comment by MrJelly17** [comment]
This is the best HA project ever. I believe that this will become a default HA dashboard in the future. Amazing

**Comment by mselmasry** [comment]
I love this ... thank you very much for everyone working on it. 
Can't wait to see what new features are coming.

I would love to contribute anyway I can, for now I am using, testing providing feedback when/where applicable. Happy to do more

**Comment by crazylobster70** [comment]
> ## 👋 Welcome!
> We’re using Discussions as a place to connect with other members of our community. We hope that you:
> 
> * Ask questions you’re wondering about.
> * Share ideas.
> * Engage with other community members.
> * Welcome others and are open-minded. Remember that this is a community we
>   build together 💪.
> 
> To get started, comment below with an introduction of yourself and tell us about what you do with this community.


Good afternoon,

First of all, let me say what a wonderful project this is — truly impressive work. However, I’ve noticed that there haven’t been any updates for a while. I was wondering whether the project is still being developed behind the scenes, or if it’s more likely that it has been discontinued.

Any insight would be greatly appreciated.
